src/vivarium_nih_us_cvd/components/risks.py

```diff
@@ -74,7 +74,30 @@ def post_processor(exposure, _):
         return post_processor
 
 
-class AdjustedRisk(DropValueRisk):
+class CorrelatedRisk(DropValueRisk):
+    """Creates risk without propensities so they can be created by correlation component"""
+
+    def __repr__(self) -> str:
+        return f"CorrelatedRisk({self.risk})"
+
+    #################
+    # Setup methods #
+    #################
+
+    def _register_simulant_initializer(self, builder: Builder) -> None:
+        builder.population.initializes_simulants(
+            self.on_initialize_simulants,
+        )
+
+    ########################
+    # Event-driven methods #
+    ########################
+
+    def on_initialize_simulants(self, pop_data: SimulantData) -> None:
+        pass
+
+
+class AdjustedRisk(CorrelatedRisk):
```

Comment:
Instead of having `CorrelatedRisk` inherit from `DropValueRisk`, you can do something like this:
```
class DropValueRisk(Risk):
    ...

class CorrelatedRisk(Risk):
    ...

class AdjustedRisk(DropValueRisk, CorrelatedRisk):
    ...

class TruncatedRisk(DropValueRisk, CorrelatedRisk):
    ...
```

The benefit of this is that the depth of inheritance is lower, but also it would leave open the ability to define a Risk class that is a `CorrelatedRisk` but not a `DropValueRisk`. It's relatively simple to understand what is going to happen in this case, since the two classes don't override the same function.

Otherwise, you might as well just combine `CorrelatedRisk` and `DropValueRisk` into a single class `CorrelatedDropValueRisk`, rather than inheriting.